orbital_core/utils/tracing.py

```diff
@@ -0,0 +1,147 @@
+import sys
+import aiotask_context
+import aiozipkin
+import logging
+from traceback import format_exception
+from typing import Set, Dict
+from aiohttp.web import HTTPException, Application, HTTPInternalServerError, Request
+from aiozipkin import APP_AIOZIPKIN_KEY, REQUEST_AIOZIPKIN_KEY
+from aiozipkin.constants import HTTP_STATUS_CODE, HTTP_ROUTE
+from aiozipkin.helpers import make_context, parse_sampled, parse_debug
+from aiozipkin.record import Record
+from aiozipkin.span import SpanAbc
+from aiozipkin.tracer import Tracer
+from aiozipkin.transport import Transport
+
+LOG = logging.getLogger(__name__)
+
+
+class TracingTransport(Transport):
```

Comment:
this transport may not be beneficial to share with the world as it emits data formatted for our internal protocol. should we keep this class internal and default the transport to something else?

it may make sense to update the signature of `bootstrap_app` to include structure for configuring the individual areas that we "preconfigure". maybe kwarg `tracing_opts` accepts a dataclass which has sensible defaults which we can override internally?

https://github.com/aio-libs/aiozipkin/blob/master/aiozipkin/transport.py

Reply:
It's a little tricky here... I could imagine the desire for other transports.

But at the same time, I'm not sure how the configuration would look in the end. We'd probably want people to plug in any arbitary Transport. as such, bootstrap_app would have to support passing in a transport object which could override the default choice.

This means we probably also need to expose similar options for the metric ingestion as well. Although, if this all integrates with OpenCensus, I would hope there's a global configuration object that we can pass to help instantiate whatever exporters the user is hoping to use in their environment.

Reply:
For now I have added support to accept transport class and kept TracingTransport as a default transport.